integration/tests/cook/test_multi_user.py

```diff
@@ -895,11 +895,11 @@ def test_user_limits_change(self):
                         self.assertEqual(resp.status_code, 200, resp.text)
 
     def test_queue_endpoint(self):
+        bad_constraints = [["HOSTNAME",
```

Comment:
This change seems unrelated?

Reply:
this test was flaky and the change seems to have exposed the flakiness

Reply:
Cook. Thanks for fixing.